crawlers/crawlers/spiders/wheyprotein_madrugaosuplementos.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+from crawlers.items import ProductItem
+from crawlers.utils import parse, product_type
+
+class WheyproteinMadrugaosuplementosSpider(scrapy.Spider):
+    name = 'wheyprotein-madrugaosuplementos'
+    store = 'madrugaosuplementos'
+    start_urls = ['http://www.madrugaosuplementos.com.br/massa-muscular/whey-protein/']
+
+    def parse(self, response):
+        products = response.css('.category-products .item')
+        for product in products:
+            yield self.parse_product(product)
+
+        next_url = self.next_page(response)
+        if next_url:
+            yield scrapy.Request(url=next_url, callback=self.parse)
+
+    def parse_product(self, product):
+        name = product.css('.product-name ::text').extract_first()
+        if product_type.is_wheyprotein(name):
+            product_price = product.css('.special-special-price span ::text').extract()
+            price = float(','.join(product_price).replace('\n,','').replace(',','.').strip())
```

Comment:
`price = float(','.join(product_price).replace('\n,','').replace(',','.').strip())`

Please, use parse.parse price(). If parser.parse price() doesn't works, fix that!